src/Controllers/Controller.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Controllers;
+
+error_reporting(E_ALL);
+
+class Controller
+{
+    private $connector;
+
+    private $connectorWithoutDatabase;
+
+    /**
+     * Controller constructor.
+     *
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+        $this->connectorWithoutDatabase = new Connector($config, null);
+        $this->connector = new Connector($config);
+    }
+
+    /**
+     * Create structure of Database with tables.
+     */
+    public function createStructure()
+    {
+        $this->createDB($this->connectorWithoutDatabase->getConnection());
+        $this->createUniversityTable($this->connector->getConnection());
+        $this->createDepartmentsTable($this->connector->getConnection());
+        $this->createStudentsTable($this->connector->getConnection());
+        $this->createHomeWorksTable($this->connector->getConnection());
+
+        header('Location: index.php');
+    }
+
+    /**
+     * Generete some data and insert them in tables.
+     */
+    public function generateData()
+    {
+        $sql = "INSERT INTO university (name,city,site) 
+				VALUES ('GeekHub', 'Cherkassy', 'www.site.com')";
+        $query = $this->connector->getConnection()->prepare($sql);
+        $query->execute();
+        header('Location: index.php');
+    }
+
+    private function createDB($connector)
+    {
+        $sql = 'CREATE DATABASE study';
+        $query = $connector->exec($sql);
+    }
+
+    private function createUniversityTable($connector)
+    {
+        $sql = 'CREATE TABLE university (
```

Comment:
university or universities? 
Use single case in all tables (university, student, department) 
or plural case, but also in all tables (universities, departments etc
You have university, but departments.